src/qt/rpcconsole.cpp

```diff
@@ -747,10 +769,12 @@ void RPCConsole::clear(bool clearHistory)
 #else
     QString clsKey = "Ctrl-L";
 #endif
-	 
+
     message(CMD_REPLY, (tr("Welcome to the %1 RPC console.").arg(tr(PACKAGE_NAME)) + "<br>" +
                         tr("Use up and down arrows to navigate history, and %1 to clear screen.").arg("<b>"+clsKey+"</b>") + "<br>" +
-                        tr("Type <b>help</b> for an overview of available commands.")) +
+                        tr("Type <b>help</b> for an overview of available commands.<br>") +
```

Comment:
Please keep the `<br>` out of the translation message, as it frequently gets lost by translators.

Reply:
I totally agree with all of this.